news/2017-08-30-new-featured-artist-oishii.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "New Featured Artist: OISHII"
+date: 2017-08-30 08:00:00 +0000
+---
+
+What do chiptunes and food have in common? They're both **OISHII**! No, really, they're the favourite things of osu!'s latest Featured Artist, blasting soon to a beatmap near you!
+
+[![](https://assets.ppy.sh/artists/17/header.jpg)](https://osu.ppy.sh/beatmaps/artists/17)
+
+A relatively new face on the chiptune scene, **OISHII** has been making waves after appearing out of literally nowhere. 
+
+All that we know is that every track of theirs is a sugar rush of 8-bit inspired hits, with video game and anime references sprinkled in to boot. Familiar with the *Kirby* games? You'll probably find **OISHII** familiar, too!
+
+Have a listen to *ONIGIRI FREEWAY* and see for yourself:
+
+<iframe style="border: 0; width: 100%; height: 120px;" src="https://bandcamp.com/EmbeddedPlayer/track=3682041395/size=large/bgcol=ffffff/linkcol=0687f5/tracklist=false/artwork=small/transparent=true/" seamless><a href="http://oishiichan.bandcamp.com/track/onigiri-freeway">ONIGIRI FREEWAY by OISHII</a></iframe>
+
+Not satisfied? Take a trip down the *COCOA WATERSLIDE*:
+
+<iframe style="border: 0; width: 100%; height: 120px;" src="https://bandcamp.com/EmbeddedPlayer/track=1332795351/size=large/bgcol=ffffff/linkcol=0687f5/tracklist=false/artwork=small/transparent=true/" seamless><a href="http://oishiichan.bandcamp.com/track/cocoa-waterslide">COCOA WATERSLIDE by OISHII</a></iframe>
+
+One thing is certain - all of **OISHII**'s tracks are explosively sweet and short, and we now have **7** of them immediately available for you to use in beatmapping.
```

Comment:
we now have **7** of them -> we currently have **7** of them, maybe?

Reply:
see where you're coming from, more of a prose-related choice there to tie into the whole 'explosively sweet' thing

like if you read it how a salesman would say it, it makes sense